src/Analyzers/MSTest.Analyzers/AnalyzerReleases.Unshipped.md

```diff
@@ -11,3 +11,4 @@ MSTEST0021 | Design | Disabled | PreferDisposeOverTestCleanupAnalyzer, [Document
 MSTEST0022 | Design | Disabled | PreferTestCleanupOverDisposeAnalyzer, [Documentation](https://learn.microsoft.com/dotnet/core/testing/mstest-analyzers/mstest0022)
 MSTEST0023 | Usage | Info | DoNotNegateBooleanAssertionAnalyzer, [Documentation](https://learn.microsoft.com/dotnet/core/testing/mstest-analyzers/mstest0023)
 MSTEST0024 | Usage | Info | DoNotStoreStaticTestContextAnalyzer, [Documentation](https://learn.microsoft.com/dotnet/core/testing/mstest-analyzers/mstest0024)
+MSTEST0025 | Usage | Info | PreferAssertFailOverAlwaysFalseConditionsAnalyzer
```

Comment:
The roslyn refactoring is broken because of the helper I did.

```suggestion
MSTEST0025 | Usage | Info | PreferAssertFailOverAlwaysFalseConditionsAnalyzer, [Documentation](https://learn.microsoft.com/dotnet/core/testing/mstest-analyzers/mstest0025)
```